app/src/androidTest/java/com/rootstrap/android/tests/ProfileActivityTest.kt

```diff
@@ -20,15 +19,15 @@ import org.junit.Test
 @HiltAndroidTest
 class ProfileActivityTest : BaseTests() {
```

Comment:
Maybe we should change the name of this class as it now validates if the MainActivity is launched successfully